app/assets/javascripts/views/project_view.js

```diff
@@ -21,6 +27,51 @@ module.exports = Backbone.View.extend({
     });
   },
 
+  prepareColumns: function() {
+    // reset columns & columns
+    this.columns = {};
+    this.$el.html(this.template({ inverse_flow: this.model.get('inverse_flow') }));
+    $('#column-toggles').html('');
+
+    var that = this;
+    $('[data-column-view]').each(function() {
+      that.addColumnView($(this));
+    });
+
+    // render columns on the screen
+    this.addAll();
+    this.scaleToViewport();
+  },
+
+  addColumnView: function(el) {
+    var data = el.data();
+    el.addClass(data.columnView + '_column');
+
+    var column = new ColumnView({
+      el: el,
+      id: data.columnView,
+      name: I18n.t('projects.show.' + data.columnView),
+      sortable: data.connect !== undefined
+    });
+
+    column.on('visibilityChanged', this.checkColumnViewsVisibility);
+    column.render();
+
+    this.columns[data.columnView] = column;
+
+    if (data.hideable !== false) {
```

Comment:
The reason is because `data.hideable` may be an empty string which is false, but it is still supposed to pass on the condition.